handbook/ops/bizops/customer_data_policy.md

```diff
@@ -1,15 +1,52 @@
+
+PII and user characteristics
+| Source/platform                                   | Sourcegraph Cloud  | On-prem/self-hosted | How long is this data kept for?                                                                      | Who can access it? |
+|---------------------------------------------------|--------------------|---------------------|------------------------------------------------------------------------------------------------------|--------------------|
+| HubSpot and Salesforce (CRM/marketing automation) | :heavy-check-mark: | :heavy-check-mark:  | Retain all customer information                                                                      |                    |
+| Communication tools (Gmail, Jira, Productboard)   | :heavy-check-mark: | :heavy-check-mark:  | Retain all inbound customer communication                                                            |                    |
+| Slack                                             | :heavy-check-mark: | :heavy-check-mark:  | Customer support channels are maintained indefinitely; all other messages are deleted after 3 months |                    |
+| View list of sales tools                          | :heavy-check-mark: | :heavy-check-mark:  | Retain all outbound customer communication                                                           |                    |
+|                                                   |                    |                     |                                                                                                      |                    |
+|                                                   |                    |                     |                                                                                                      |                    |
+|                                                   |                    |                     |                                                                                                      |                    |
+
+
+## FAQs
+
+### Where is my data stored?
+
+All outside tools are cloud products and therefore the data is hosted by the service provider. All internal tools and product data is stored in the Postgres database that hosts Sourcegraph Cloud, as well as data pipelines and warehouses in Google Cloud Platform. 
+
+### Can Sourcegraph developers or admins see my private code?
+
+No. Only you can see or edit your private code. Code host permissions are enforced and not even Sourcegraph’s admin accounts can see private code. Manage your permissions in the code host and they will be automatically replicated in Sourcegraph.
+
```

Comment:
@ryphil we should also answer this: 'Can Sourcegraph developers or admins see my private settings and Sourcegraph configuration?' 

I think the answer is yes, because we access Postgres and therefore can see their saved searches for example, but are doing a lot around limiting access to core/distro teams.

Reply:
[Slack convo I started](https://sourcegraph.slack.com/archives/C022H6ZQD7S/p1625841051065900)

Reply:
@ebrodymoore - From the slack thread and Ryan/Joe's reply, it looks like that was an oversight and should not be the case. Thus will be fixed prior to GA and we can leave the answer as 'no'?